charts/internal/controlplane-shoot/charts/udp-egress-dummy/templates/udp-egress-dummy.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if .Values.enabled }}
+# this dummy service creates a LoadBalancer with a UDP port to enable outgoing UPD traffic for Shoots using
+# Azure Standard LoadBalancers (see https://github.com/gardener/gardener-extension-provider-azure/issues/1)
+# TODO (dkistner): this workaround should be removed once we have a proper implementation for UDP egress via NAT GW
+apiVersion: v1
+kind: Service
+metadata:
+  name: udp-egress-dummy
+  namespace: kube-system
+  labels:
+    app: dummy
```

Comment:
Can you add an annotation and explain why we create this? People might see this service/try to delete it.